api/swagger.yaml

```diff
@@ -6238,6 +6238,7 @@ paths:
             - `id=<network-id>` Matches all or part of a network ID.
             - `label=<key>` or `label=<key>=<value>` of a network label.
             - `name=<network-name>` Matches all or part of a network name.
+            - `scope=["swarm"|"local"]` Filters networks by scope (either `swarm` or `local`).
```

Comment:
Thanks @cpuguy83 @thaJeztah. Yes I forgot libnetwork have the legacy external K-V store mode (`global`).

The PR has been updated and rebased. Please take a look.